### PR TITLE
[OSDEV-896] Implement Sector serializer to parse sector, product_type, sector_product_type fields

### PR DESCRIPTION
**Pull request by mazursasha1990**
[OSDEV-896](https://opensupplyhub.atlassian.net/browse/OSDEV-896)
Implemented functionality for RowSectorSerializer that parses sector, product_type, and sector_product_type fields.

### CODE REVIEW

**Review by killev — DISMISSED** [review]
Looks good. Approved!

File: src/django/api/sector_product_type_parser.py [hunk 1]
```diff
@@ -37,7 +38,7 @@ def __init__(self):
 
     def parse_all_values(self, all_values):
         sector_map = SectorProductTypeParser.sector_cache.sector_map
-
+        print('sector_map:', sector_map)
```
Comment (roman-stolar) [inline src/django/api/sector_product_type_parser.py:41]: please remove **print** function

**Review by roman-stolar — CHANGES_REQUESTED** [review]
Minor comment, in genreal LGTM, waiting for changes and approve.

File: src/django/api/sector_product_type_parser.py (hunk 1 above)
Comment (mazursasha1990) [reply src/django/api/sector_product_type_parser.py]: Done

**Review by roman-stolar — APPROVED** [review]
LGTM

**Review by killev — APPROVED** [review]
Looks good. Approved

**Pull request merged**